docker-wrapper.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/sh
+
+# Variables setting
+HOST=`hostname -i`
+petalscli="/opt/petals-deployer/bin/petals-cli.sh -h $ENV_CLI_HOST -n $ENV_CLI_PORT -u $ENV_CLI_USER -p $ENV_CLI_PASS -y -c "
+
+# Informational dump
+echo "############################ PETALS DEPLOYER ############################"
+echo "###     Starting deploy script with following parameters:" 
+echo " - petals host: $ENV_CLI_HOST" && echo " - petals port: $ENV_CLI_PORT" && echo " - petals user: $ENV_CLI_USER" && echo " - petals pass: $ENV_CLI_PASS"
+echo " - deploy file: $ENV_DEPLOY_FILE"
+echo " - URLS to deploy: $ENV_DEPLOY_URLS"
+echo " - CLI host: ${HOST}"
+echo " - CLI version :"
+/opt/petals-deployer/bin/petals-cli.sh --version
+
+# CLI environment file setting
+echo ""
+echo "###     Setting configuration..." 
+# For now this deployer must be used without "--net=host" option
+echo "#docker-petals-deployer artifacts server configuration" >> /opt/petals-deployer/conf/petals-cli.default 
+echo "embedded.http.port=80" >> /opt/petals-deployer/conf/petals-cli.default 
+echo "embedded.http.host=${HOST}" >> /opt/petals-deployer/conf/petals-cli.default
+echo "" >> /opt/petals-deployer/conf/petals-cli.default 
+
+# Any point using an alias ?
+# echo "#docker-petals-deployer petals instance target alias" >> /opt/petals-deployer/conf/petals-cli.default 
+# echo "deployertarget.host=$ENV_CLI_HOST" >> /opt/petals-deployer/conf/petals-cli.default
+# echo "deployertarget.port=$ENV_CLI_PORT" >> /opt/petals-deployer/conf/petals-cli.default
+# echo "deployertarget.username=$ENV_CLI_USER" >> /opt/petals-deployer/conf/petals-cli.default
+# echo "deployertarget.password=$ENV_CLI_PASS" >> /opt/petals-deployer/conf/petals-cli.default
+# echo "" >> /opt/petals-deployer/conf/petals-cli.default 
+# petalscli="/opt/petals-deployer/bin/petals-cli.sh -y -a deployertarget -c "
+
+echo ""
+echo "###     Target Petals ESB container status:"
+echo ""
+$petalscli version
+$petalscli topology-list
+echo "Artifacts :"
+$petalscli list
+echo ""
+$petalscli endpoint-list
+
+echo ""
+echo "###     Starting deploy ..."
+
+# Deploy artifacts from urls
+if [ ! -z ${ENV_DEPLOY_URLS+x} ] && [ ! "$ENV_DEPLOY_URLS" == "" ] 
+then
+  echo ""
+  echo "# URLs found, deploying:"
+  IFSBACKUP="$IFS"
+  IFS=';'
+  set $ENV_DEPLOY_URLS
+  IFS="$IFSBACKUP"
+  for url in $*
+  do
+    echo "Deploying $url..."
+    $petalscli deploy -- -u $url
+  done
+  echo "# URLs deploy over."
+else
+  echo "No URLs found."
+fi
+
+# Deploy artifacts from file
+if [ ! -z ${ENV_DEPLOY_FILE+x} ] && [ ! "$ENV_DEPLOY_FILE" == "" ] 
+then
+  echo ""
+  echo "# File found, deploying:"
+  while IFS='' read -r file || [[ -n "$file" ]]; do
```

Comment:
$file should be called $line, shouldn't it?

Reply:
Both felt right to me, I put `$file` as the previous deploying block uses `$urls`.
But actually we could be passing urls or files in either block so I'll change to something like `$line` and `$item`